packages/widget/README.md

```diff
@@ -0,0 +1,55 @@
+# Sygmaprotocol Lit Widget
+
+### Dependencies
+ 
+To integrate the Widget to any lit project that you have, you will need to add the following dependencies:
+
+```bash
+yarn add @buildwithsygma/sygmaprotocol-widget @buildwithsygma/sygma-sdk-core
```

Comment:
why do you need sdk? Doesn't it come installed with the widget?

Reply:
I was thinking here about defining `environment` property. We don't expose this on the widget, and we are using it from the SDK.